src/Ledger/GovernanceActions.lagda

```diff
@@ -131,18 +243,44 @@ record GovProposal : Set where
         prevAction  : NeedsHash action
         anchor      : Anchor
 \end{code}
-\begin{code}[hide]
+\caption{Governance action proposals and votes}
+\label{defs:governance-votes}
+\end{figure*}
+\emph{Any Ada holder} can submit a governance action to the chain.
```

Comment:
We usually don't mention 'Ada' and 'Lovelace' in the spec. This is also just a marketing line that we don't need to repeat.

Reply:
Okay, I removed references to Ada and Lovelace, but kept the explanation about the deposit (required for submitting action to the chain) that will be refunded later... or did you want me to delete that too?

For the same reason I should remove the paragraph,

"As with stake pools, any Ada holder may register to be a DRep and so choose to represent themselves and/or others. Also, as with stake pools, Ada holders may instead delegate their voting rights to any other DRep. Voting rights will be based on the total Ada that is delegated, as a whole number of Lovelace."

that appears near the start of this section, correct?